templates/wizard.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends '@Tabler/fullpage.html.twig' %}
+
+{% block page_content %}
+
+    <div
+            class="container {% block container_size %}container-narrow{% endblock %} {% block container_class %}py-4{% endblock %}"
+            {% block container_attr %}{% endblock %}
+    >
+        {% block wizzard_before %}{% endblock %}
+
+        {% block wizzard_logo %}
+            <div class="text-center mb-4">
+                <h1 class="navbar-brand navbar-brand-autodark mb-4">
+                    <a href="{{ path('tabler_welcome'|tabler_route) }}">
+                        {% if tabler_bundle.getLogoUrl() is not empty %}
+                            <img src="{% if '://' in tabler_bundle.getLogoUrl() %}{{ tabler_bundle.getLogoUrl() }}{% else %}{{ asset(tabler_bundle.getLogoUrl()) }}{% endif %}"
```

Comment:
Having an empty `alt=""` might be better than a default string

Reply:
Yes, good idea!

Reply:
Logo code is now under `embed` => [/templates/embeds/logo.html.twig]()